extensions/3rdparty/RSHighscores/RSHighscores.body.php

```diff
@@ -113,7 +113,7 @@ private static function parseHiscores( $data, $skill, $type ) {
 	 * @param int $type Index representing the requested type of data for the given skill.
 	 * @return string
 	 */
-	public static function renderHiscores( &$parser, $hs = 'rs3', $player = '', $skill = -1, $type = 1 ) {
+	public static function renderHiscores( $parser, $hs = 'rs3', $player = '', $skill = -1, $type = 1 ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)